.github/workflows/release-helm-chart.yaml

```diff
@@ -0,0 +1,53 @@
+# Description: This workflow is used to release the Helm chart to the GitHub repository. The chart manifests should be already
```

Comment:
https://github.com/Kuadrant/kuadrant-operator/pull/728/files#r1692800237

Reply:
These workflows are triggered by the publishing of a release or the deletion of one, so the kuadrant.io/helm-charts chart repo can update its index accordingly. It can also be triggered by a workflow_dispatch event.

Reply:
> For the release of the package, it will be tested once merged with alpha pre releases.

Grabbing this from the PR description, because it appeared I missed it on reading! 

From what I gather, this workflows will be tested when this PR is merged. I guess when dealing with uploading/deleting files for releases it can be tough to test until it is on `main` to test . 

OK, no problem. I will resolve for now and take a look further below to review new changes.